repos/system_upgrade/el7toel8/actors/mariadbcheck/libraries/library.py

```diff
@@ -0,0 +1,227 @@
+from leapp.models import InstalledRedHatSignedRPM
+from leapp.libraries.common import reporting
+from leapp.libraries.stdlib import CalledProcessError, api, run
+from os import listdir
+from os.path import isfile, join, isdir
+import re
+
+
+def get_config_files(rootfile):
```

Comment:
There is a binary called `my_print_defaults` that parses the config files and handles the other directories via `!include` directive. I think we should use that binary. However, this does not need to block this PR.